tests/unit/setup.js

```
@@ -192,6 +192,13 @@ global.createModuleStore = (vuexModule, options = {}) => {
 };
 */
 
+// ===
+// Let tests fail on console.error
+// ===
+console.error = jest.fn((error) => {
+	throw `💔 ${error}`;
+});
```

We had some tests that we logging errors to the console, but are not failing. This trick converts all error logs into real errors and let the test fail.

---

Nice I was wondering about that while looking at travis earlier :)